config/webpack.prod.js

```diff
@@ -39,6 +41,7 @@ const PUBLIC_PATH = process.env.PUBLIC_PATH || '/';
 const BUILD_NUMBER = process.env.BUILD_NUMBER;
 const BUILD_TIMESTAMP = process.env.BUILD_TIMESTAMP;
 const BUILD_VERSION = process.env.BUILD_VERSION;
+const FABRIC8_BRANDING = process.env.FABRIC8_BRANDING || 'fabric8';
```

Comment:
Why aren't we using the OS icon here in prod? You have it provisioned but don't use it.